src/averageAges.js

```diff
@@ -37,7 +46,12 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(person => person.sex === 'f');
+  const womenWithChildren = women.filter(person => people.find(
+    child => child.mother === person.name
+  ));
```

Comment:
Let's use one filter instead of two, we can select needed condition to return inside one filter method

Reply:
I have two different variables which need for task

Reply:
So you've filtered array twice, but you can do it only once

Reply:
@AniaHna fix it